reqs/tests/link_agencies_tests.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from model_mommy import mommy
+
+from reqs.management.commands import link_agencies
+from reqs.management.commands.sync_agencies import Command as SynchCommand
```

Comment:
I've got the most enjoyable dictionary on twitter on my side: https://www.merriam-webster.com/dictionary/sync

Doesn't matter to me, so long as our docs/scripts are up to date :)